packages/dataviews/src/field-types/text.tsx

```diff
@@ -49,6 +53,18 @@ function Edit< Item >( {
 		[ id, onChange ]
 	);
 
+	if ( field.elements && field.editAs === 'radio' ) {
```

Comment:
I think we need to remove all these conditions and have a folder for control and something like:

```
const Control = getFieldControl( field )

<Control />
```